ONX.CRM.BLL/Services/GroupService.cs

```diff
@@ -12,5 +16,60 @@ public GroupService(IRepository<Group> repository) : base(repository)
         {
             
         }
+
+        public async Task<IEnumerable<Group>> SearchGroups(string query, string status)
```

Comment:
Two methods SearchByQuery and SearchByStatus are preferable to use here

Reply:
done